packages/platform-sdk-profiles/src/drivers/memory/wallets/wallet.synchroniser.ts

```diff
@@ -12,15 +12,17 @@ export class WalletSynchroniser implements IWalletSynchroniser {
 	/** {@inheritDoc IWalletSynchroniser.coin} */
 	public async coin(options: { resetCoin: boolean } = { resetCoin: false }): Promise<void> {
 		if (options.resetCoin) {
-			this.#wallet.getAttributes().set(
-				"coin",
-				this.#wallet.profile()
-					.coins()
-					.push(this.#wallet.coinId(), this.#wallet.networkId(), {}, true),
-			);
+			// @TODO: create new instance without custom configuration
```

Comment:
This is a bit of an odd one because we would need to create a new instance if a reset of the coin is requested but we want to avoid to create coin instances on our own.